back/app/serializers/web_api/v1/static_page_serializer.rb

```diff
@@ -17,5 +53,9 @@ class WebApi::V1::StaticPageSerializer < WebApi::V1::BaseSerializer
     current_navbaritem_title || NavBarItem.new(code: 'custom', static_page: object).title_multiloc_with_fallback
   end
 
+  has_many :pins
+  has_many :pinned_admin_publications
```

Comment:
Good point, that I had totally not considered! Also, when we do apply permissions, this may well mean the user has no right to view any of the 3 `pinned_admin_publications`

Reply:
We decided we should deal with permissions issues when we get to implementing the static_page projects list.

Reply:
@alexander-cit The acceptance test for updating pins fails without the `has_many :pinned_admin_publications` here.

Reply:
Ok then. As we've been discussing it quite a lot, I hope we won't forget it.